### DISCUSSION TITLE
Add a Mailbox in 1 step

### DISCUSSION THREAD

**Discussion by rpc31** [simple-login/app#413] [Feature Requests]
I am not asking if there is a way to remove the verification of a mailbox, I understand how important it is to verify them. Just the ability to “add them” to an alias even if they haven’t been verifier yet. 

In the alias, they would appear as “inactive/not verified”. And it would become active automatically as soon as it is activated from the user side. But this way we do not have to re-connect and check every time a user is activating its mailbox in order to, then only, being ablate add it.

**Comment by dj-le-github** [comment]
Actually, is there a way to support forwarding without verifying?  We have use case where we create a forwarding email for each one of our clients to standardize the email format.  For example, every client will have an email format like this:  firstname.lastnameXXXXX@mydomain.com where XXXXX is their client ID that we gave them.  The only thing that this standardize email does is to forward to the client's real email.   So, for example, we have John Doe with client ID 12345 with his email being johndoe@gmail.com.  Then we would like to create a standardize email with our domain as john.doe12345@mydomain.com that simply forwards all email sent to this email (john.doe12345@mydomain.com) to John Doe's real email at johndoe@gmail.com.

In this case, we don't want to verify the email because it is forwarding to our client, and we don't them to have to verify simply because we created this forwarding email to make it easier for us to standardize the email format so we can email them easier.

**Reply by nguyenkims** [reply]
@dj-le-github You can add a new endpoint in SimpleLogin API to mark a mailbox as verified. The api doc is on https://github.com/simple-login/app/blob/master/docs/api.md. This endpoint should be quite straightforward and can be used in your custom script. It can also be part of the `POST /api/mailboxes` endpoint.
